azext_edge/edge/providers/check/common.py

```diff
@@ -327,4 +327,16 @@ class DataLakeConnectorTargetType(ListableEnum):
 
 MIN_K8S_VERSION = "1.20"
 
+# Node prerequisite constants
+
+MIN_NODE_MEMORY = "16G"
+MIN_NODE_STORAGE = "30G"
+MIN_NODE_VCPU = "4"
+AIO_SUPPORTED_ARCHITECTURES = ["amd64", "arm64"]
```

Comment:
arm64? Can you share the reference that arm64 is supported ?

Reply:
wat... ok will change as more comments come in